src/types/domain/Player.ts

```diff
@@ -3,6 +3,7 @@ import { Match } from "./Match";
 export type Player = {
     name: string;
     matches: Match[];
```

Comment:
can we add some JSDOC here now to explain the difference between matches and validMatchCount? 

something like:

a collection of Match objects that the player has participated in. This includes all matches regardless if they are valid or not.

Reply:
Done.